cli/src/main/kotlin/commands/UploadResultToSw360Command.kt

```diff
@@ -98,11 +98,11 @@ class UploadResultToSw360Command : CliktCommand(
                 val name = listOfNotNull(pkg.id.namespace, pkg.id.name).joinToString("/")
```

Comment:
Commit message nit: The prefix "cli" is a bit unspecific. Can you use the class name or is this too long? Otherwise, I would suggest something like "SW360 upload:".

Reply:
What about _cli: Replace orElse with orElseGet in UploadResultToSw360Command_?

Reply:
Also fine.